scripts/migrate_project_mailing_lists.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+"""Create mailing lists for all top-level projects
+"""
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+
+from website.app import init_app
+from website.mailing_list.utils import create_list as real_create_list
+from website.models import Node
+from website.notifications.model import NotificationSubscription
+from website.notifications.utils import to_subscription_key
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+EVENT = 'mailing_list_events'
+SUBSCRIPTION_TYPE = 'email_transactional'
+
+def fake(*args, **kwargs):
+    # Override actual method to prevent outgoing calls
+    return
+
+def get_targets():
+    return Node.find()
```

Comment:
> That part of the spec was thrown out. It should apply to every node that isn't registered, deleted, or a collection.

Good to know. Clarifying the docstring might be helpful- but there's still a wide gulf.

```
>>> Node.find(utils.PROJECT_QUERY).count()
6251
```